erpnext/public/js/education/lms/quiz.js

```diff
@@ -21,7 +21,7 @@ class Quiz {
 
 	make(data) {
 		data.questions.forEach(question_data => {
-			let question_wrapper = document.createElement('div');
+			let question_wrapper = document.createElement('li');
```

Comment:
Please add screenshots to your PR to show the impact of this and other changes made.

Reply:
done

Reply:
done